### ISSUE TITLE
Tweak to presentation of results 

### ISSUE THREAD

**Issue by conovermitch** [OHDSI/OhdsiShinyModules#325]
**Issue**: The Covariate Threshold feature within the Database Comparison interface is being misinterpreted by users. Users of the dashboard interpret the "< min threshold" to mean N<5.  However, it actually reflects that the percentage column is below the Covariate Threshold set using the slider interface. This can be confusing for a number of reasons: 1) When the Covariate Threshold slider is set to 0 the minimum threshold applied to the table still appears to be 1%. Ideally, the user would not have the option of selecting thresholds below the minimum possible threshold.  2) The "<min threshold" dialog only appears when there is some cell with populated contents. In instances where all columns are "<min threshold" for a given row the entire row is censored from the data.  3) The table can show counts <5 as long as the "min threshold" (which is on the proportion scale) has been met.  

Please note that all of this behavior is different from the corresponding functionality in the Incidence Rate Table which appears to use the notation "-5" when a cell count is less than 5. I'm not necessarily implying that the functionality should be the same for the two tables but I do think some of the ambiguity results in misinterpretation of the data. 

**Location of issue**: Characterization module --> Cohort Summary --> Database Comparison  --> Covariate Threshold

**Package Versions**: Generated with OhdsiShinyModules v2.1.5.9000 and ShinyAppBuilder v2.0.1.9000

**Comment by jreps** [comment]
Yeah - this is also linked to the FE setting that specifies minimum frequency.  I have that easy to access so should be able to edit the column to have < a value rather than < min threshold.  Will add this as an improvement to make.
